Up2dateService/Up2dateClient/Client.cs

```diff
@@ -199,10 +180,24 @@ private void OnDeploymentAction(IntPtr artifact, DeploymentInfo info, out Client
                     case InstallPackageStatus.PsScriptInvokeError:
                         additionalMessage = "General Choco Error";
                         break;
+                    case InstallPackageStatus.MsiInstallationError:
+                        additionalMessage = "Msi Installation Error";
+                        break;
+                    case InstallPackageStatus.InstallationPackageIsNotSigned:
+                        additionalMessage = "File not signed. File deleted";
+                        File.Delete(filePath);
+                        break;
+                    case InstallPackageStatus.InstallationPackageIsNotSignedBySelectedIssuer:
+                        additionalMessage = "File not signed by selected issuer. File deleted";
```

Comment:
Suggestion:
"Package is not signed by any whitelisted issuer. Deployment rejected"

Reply:
Done